docs/sbctl.8.txt

```diff
@@ -113,12 +113,28 @@ EFI binary commands
                 Sign all the generated bundles.
 
 **remove-bundle** <NAME>::
-        Removes a bundle from the list.
+        Removes a bundle from the list. This does not delete the bundle itself.
 
 **list-bundles**::
         List all registered bundles to generate.
 
 
+Bundles
+-------
+
+Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)
+into a single file which can be easily signed. They also don't read parameters
+from external files, which makes them more tamper-proof.
+
+When a bundle is generated, its metadata is stored into the bundle database
```

Comment:
Not sure if I prefer `metadata` or `manifest` here. Thoughts?

Reply:
`configuration` is better I think.